gradle-baseline-java/src/main/groovy/com/palantir/baseline/extensions/BaselineErrorProneExtension.java

```diff
@@ -48,6 +48,7 @@
             "ObjectsHashCodeUnnecessaryVarargs",
             "OptionalFlatMapOfNullable",
             "OptionalOrElseMethodInvocation",
+            "PatternAsKeyOfSetOrMap",
```

Comment:
This line can be removed, it's only necessary for checks we want to automatically apply suggested fixes for. Mind adding a line to the readme describing the check instead?